src/topology/algebra/infinite_sum.lean

```diff
@@ -474,6 +494,19 @@ end
 
 end has_continuous_add
 
+section has_continuous_star
+variables [star_add_monoid α] [has_continuous_star α]
+
+lemma tsum_star : star (∑' b, f b) = ∑' b, star (f b) :=
```

Comment:
Perhaps I'm missing something obvious, but why can't this go directly below the other new star summable lemmas?

Reply:
It's mainly to match how the existing `has_continuous_add` lemmas are split. This lemma requires a `t2_space`, the ones above don't.